common/src/main/java/org/apache/uniffle/common/ShufflePartitionedData.java

```diff
@@ -71,4 +77,8 @@ public ShufflePartitionedBlock[] getBlockList() {
   public long getTotalBlockSize() {
     return totalBlockSize;
   }
+
+  public long getTotalBlockLength() {
```

Comment:
What's the difference of block lenght and block size?

Reply:
@zuston 

```java
  public int getSize() {
    return length + 3 * 8 + 2 * 4;
  }
```

The block size is larger  `3 * 8 + 2 * 4` than block length which instead of encoded length of this block.